apps/codeforafrica/src/pages/projects/[slug].page.js

```diff
@@ -16,9 +17,20 @@ import TeamMemberCardList from "@/codeforafrica/components/TeamMemberCardList";
 import { projects, getPageStaticProps } from "@/codeforafrica/lib";
 
 function Index({ project, sections, ...props }) {
+  const { badges } = project;
   return (
     <Page {...props}>
       <ProjectPageHeader {...project} />
+      <Section
+        sx={{
+          marginTop: { xs: "26.6px", sm: "20px", md: "56px" },
+          marginBottom: "42px",
+          paddingRight: { xs: "20px", md: 0 },
+          paddingLeft: { xs: "20px", md: 0 },
```

Comment:
```suggestion
          px: { xs: 2.5, sm: 0 },
```

x => left and right.